cli/src/pcluster/schemas/common_schema.py

```diff
@@ -66,6 +66,19 @@ def only_one_field(self, data, field_list, **kwargs):
             return True
         return len([data.get(field_name) for field_name in field_list if data.get(field_name)]) == 1
 
+    def fields_coexist(self, data, field_list, **kwargs):
```

Comment:
This has the same code of the `only_one_field`, except for the `>` operator.
Let's select one of the two methods and use it everywhere.

I like the fields_coexist one, let's replace the old one.